test/e2e/data/infrastructure-docker/main/cluster-template-topology-dualstack-ipv4-primary/cluster.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: '${CLUSTER_NAME}'
+spec:
+  topology:
+    class: quick-start
+    variables:
+       - name: ipv6Primary
+         value: false
+       - name: externalCloudProvider
+         value: true
```

Comment:
While it is intriguing to making progress towards having a "CAPD" external cloud provider, I'm wondering if it could help us to make a first iteration were we focus on the bare metal use case first (where there is no cloud provider at all)

Reply:
I think the implementation of the cloud provider is simple enough - and the alternative is to do a lot of hacking around the kubeadm bootstrap in CAPD so this way feels cleaner IMO.